src/app/files/fileProvider.js

```diff
@@ -63,15 +66,23 @@ class FileProvider {
     return true
   }
 
-  addReadOnly (path, content) {
+  addReadOnly (path, content, url) {
+    this.readonlyItems.push(path)
```

Comment:
we should keep this in the local storage. Perhaps by having a `.remix-system` file.

Reply:
next PR for that?

Reply:
yes